client/src/context/useTeamBoardContext.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState, useContext, createContext, FunctionComponent } from 'react';
+
+interface ITeamBoardContext {
+  teams: any;
+  activeTeam: any;
+}
+
+export const TeamBoardContext = createContext<ITeamBoardContext>({
+  teams: '',
+  activeTeam: '',
+});
+
+export const TeamBoardProvider: FunctionComponent = ({ children }): JSX.Element => {
+  const [teams, setTeams] = useState<string>('');
+  const [activeTeam, setActiveTeam] = useState<string>('');
```

Comment:
Users will have multiple teams, and there will be more data than just string. This should be refactored to be an array of Team types

Reply:
Another point too, you are using immer which is great and you got some actions setup. But I don't even see that being used here?

Reply:
:) Certainly the data type of the teams and an active team are going to be much more than a string, but I left it that way, because we are not totally sure of the fields `Team` will have I am also discussing with the team .

That's great advice Ethan, I only thought about `useImmerReducer` to dispatch the actions, because it was mentioned in the ticket, but it makes more sense to also use `useImmer` to set the initialState I will be making that change. I will also go with my instincts and type both `activeTeam` and `teams` we can make modifications to it while integrating if need for it arises.

Reply:
Just a hint @Robertito1 in typescript when you set for example empty quotes the typescript already got the typo. So you don't need to type again like <string>('') if you set just ('') the typescript already knows which type is.

Reply:
Thanks for the heads up Igor, that code is outdated now though because I just created a `Teams` and `activeTeam` interface and that is what I used for the initial state not the string